app/src/ccdashboard/client/core.cljs

```diff
@@ -86,6 +88,42 @@
                            :component-did-mount  (partial progress-did-mount state-atom component-name stat-key total-stat-key format-fn)
                            :component-did-update (partial progress-update state-atom component-name stat-key total-stat-key format-fn)})))
 
+(defn location-stats [state-atom component-name]
+  [:div {:style {:margin-left  "auto"
+                 :margin-right "auto"}
+         :id    component-name}
+   [:svg]])
+
+(defn location-stats-did-mount [state-atom component-name]
+  (.addGraph js/nv (fn []
+                     (let [state @state-atom
+                           chart (if (> 992 (get-in state [:viewport/size :width]))
```

Comment:
a `(def ^:const location-stats-orientation-switch-threshold 992)` or something like that would make the magic number less magic